### DISCUSSION TITLE
nemo 1.0.0rc1 docker is ready to use

### DISCUSSION THREAD

**Discussion by asher-bs** [NVIDIA-NeMo/NeMo#2060] [Show and tell]
I made this image with nemo version 1.0.0rc1
https://hub.docker.com/r/docker4b/nvidia_nemo_1.0.0rc1

To run it: 
docker run --gpus all -it --rm   -w /workspace  --shm-size=8g -p 8888:8888 -p 6006:6006 -p 5000:5000 --ulimit memlock=-1 --ulimit stack=67108864 --device=/dev/snd  docker4b/nvidia_nemo_1.0.0rc1:latest

**Comment by okuchaiev** [comment]
Thanks for sharing! Could you please give more details on what is it based on?

**Reply by asher-bs** [reply]
it is based on the official docker

**Comment by titu1994** [comment]
Thank you for creating the container.

The official nemo docker is available on NGC - `nvcr.io/nvidia/nemo:1.0.0rc1`
It can be found at - https://ngc.nvidia.com/catalog/containers/nvidia:nemo

**Reply by asher-bs** [reply]
This official docker in not full, it doesn't include Pyaudio
